misc/profiles2/trekking.brf

```diff
@@ -307,6 +433,10 @@ assign footaccess =
        else if foot= then defaultaccess
        else not foot=private|no
 
+assign trafficsignalcost =
```

Comment:
I added an extra cost for each traffic signal. It does not change much the routes found by BRouter (as traffic signals repartition is rather homogeneous and isotropic I guess), but helps in some particular cases where the cycleway is poorly designed and forces to wait at several traffic lights in a row.

There is such an example in the test suite where the cycleway forces to go downhill, pass 3 traffic signals and go back uphill while you could actually just use the pedestrian crossing (with a foot penalty) and only wait for one traffic signal.

Reply:
https://groups.google.com/forum/#!msg/osm-android-bikerouting/plmpWPPcqqQ/QqfpEKr5AwAJ is a discussion of interest about traffic signals penalty.